types/passport/index.d.ts

```diff
@@ -12,7 +12,7 @@ declare global {
     namespace Express {
         interface Request {
             authInfo?: any;
-            user?: User;
+            user?: any;
```

Comment:
What's the point of types if it's just an any? This should be generic if at all possible.